docs/sources/query/query_accceleration.md

```diff
@@ -26,6 +26,11 @@ If [bloom filters][] are enabled, you can write LogQL queries using [structured
 Queries will be accelerated for any [label filter expression][] that satisfies _all_ of the following criteria:
 
 * The label filter expression using **string equality**, such as `| key="value"`.
+    * `or` and `and` operators can be used to match multiple values, such as `| detected_level="error" or detected_level="warn"`.
+    * _Basic_ regular expressions are automatically simplified into a supported expression:
+        * `| key=~"value"` is converted to `| key="value"`.
+        * `| key=~"value1|value2"` is converted to `| key="value1" or key="value2"`.
+        * `| key=~".+"` checks for existence of `key`. `.*` is not supported.
```

Comment:
Should we add here a bullet for op concat such ads `value[0-9]`?

Reply:
I can. We really only support it because of how the regex parser simplifies some expressions, so it's more of an internal detail rather than a deliberate choice.

If we document it, I fear it becomes part of the design forever and we won't be able to remove it later.  

WDYT? @salvacorts @chaudum

Reply:
I would not document it

Reply:
> I fear it becomes part of the design forever
Good call. Happy to leave it out then 👍